src/runners/http_service_runner.ts

```diff
@@ -66,6 +70,19 @@ export async function runHttpServiceAsync(
     // SRA http service
     app.use(SRA_PATH, createSRARouter(dependencies.orderBookService));
 
+    // Meta transaction http service
+    if (dependencies.metaTransactionService) {
+        app.use(META_TRANSACTION_PATH, createMetaTransactionRouter(dependencies.metaTransactionService));
+    } else {
+        logger.error(`API running without meta transactions service`);
+    }
+    const signerService = new SignerService();
+    const handlers = new SignerHandlers(signerService);
+    app.post(
```

Comment:
Did we want this generally available on all HTTP services or just on certain runners?

Reply:
So, it would be great to get clarity from you on this. I made it available on both the `http_service_runner` and `signer_runner`. I was under the impression that `http_service_runner` is used when testing locally, and the individual runners are spun up in production. Is that correct?

Reply:
`index.ts` is ran as a default "everything", used for Launch Kit and local to bring everything up. 

`http_service_runner` is staking/sra/swap quote which we run separately and scale up. So if it lands in here it will be in every available HTTP endpoint.

If we're limiting this endpoint to just 1 then it shouldn't be in `http_service_runner`.